src/data/links/2020-04-05_korea_green_film_festival.yaml

```diff
@@ -0,0 +1,12 @@
+title: Green Film Festival in Seoul
+url: 'http://www.greenfund.org/en/m61.php'
+countries: [kr]
+categories:
+  - festival
```

Comment:
These categories are a small defined list in src/categories. You can invent any tags you like, but categories should be from there. :)

Reply:
got it. Hard to find in the error logs! Will correct.